.github/workflows/argus-docker-build.yaml

```diff
@@ -60,6 +60,7 @@ jobs:
         with:
           filterOutClosed: true
           filterOutDraft: false
+          sha: ${{ github.event.pull_request.head.sha }}
```

Comment:
If this is triggered by a push `github.event.pull_request.head.sha` won't have a value, right? Will that cause any issues?

Reply:
I tested it with a push and it still works